modular_skyrat/modules/altborgs/code/robot_model.dm

```diff
@@ -85,6 +85,7 @@
 		"ValeDark" = list(SKIN_ICON_STATE = "valeservdark", SKIN_ICON = CYBORG_ICON_SERVICE_WIDE, SKIN_FEATURES = list(R_TRAIT_UNIQUEWRECK, R_TRAIT_WIDE)),
 		"Partyhound" = list(SKIN_ICON_STATE = "k69", SKIN_ICON = CYBORG_ICON_SERVICE_WIDE, SKIN_FEATURES = list(R_TRAIT_UNIQUEWRECK, R_TRAIT_WIDE)),
 		"Borgi" = list(SKIN_ICON_STATE = "borgi-serv", SKIN_ICON = CYBORG_ICON_SERVICE_WIDE, SKIN_FEATURES = list(R_TRAIT_UNIQUEWRECK, R_TRAIT_WIDE, R_TRAIT_SMALL)),
+		"Vix" = list(SKIN_ICON_STATE = "vixserv", SKIN_LIGHT_KEY = "eyes-vixserv", SKIN_ICON = CYBORG_ICON_SERVICE_WIDE, SKIN_FEATURES = list(R_TRAIT_UNIQUEWRECK, R_TRAIT_WIDE))
```

Comment:
Also, that's a minor pet-peeve of mine, but could you rename the icon_states in the icon file to be `vix_serv` (and just `vix_[departmenthere]`) instead of `vixserv`, for all of them? It's easier to read, especially in the editor.
```suggestion
		"Vix" = list(SKIN_ICON_STATE = "vix_serv", SKIN_LIGHT_KEY = "eyes-vix_serv", SKIN_ICON = CYBORG_ICON_SERVICE_WIDE, SKIN_FEATURES = list(R_TRAIT_UNIQUEWRECK, R_TRAIT_WIDE)),
```